charts/backingservices/charts/srs/templates/srsservice_deployment.yaml

```diff
@@ -105,6 +105,10 @@ spec:
               value: ""
           resources:
             {{- toYaml .Values.srsRuntime.resources | nindent 12 }}
+          securityContext:
```

Comment:
do we need the securityContext key if `Values.srsRuntime.securityContext` is not provided in values.yaml? 
Can you please add unit tests for the same?